client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,66 @@
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import useStyles from './useStyles';
+import React from 'react';
+import Calendar from './Calendar';
+import NextBookings from './NextBookings';
+import ManageBookings from './ManageBookings';
+
+export default function Bookings(): JSX.Element {
+  const classes = useStyles();
+  // temp {} to hold data from API request.
```

Comment:
Hey, you can remove this comment,

Reply:
hi @ajaytomgeorge. yes I can remove it, but the reason it's there is to leave a note when this UI gets integrated with the BE. I will take this recommendation for future commits .